lib/net/vnc.rb

```diff
@@ -263,6 +263,19 @@ def close
       socket.close
     end
 
+    def reconnect
+      60.times do
```

Comment:
Why 60 times?

Reply:
there's not specially mean, just followed `clipboard` method.

```
    def clipboard
      if block_given?
        @clipboard = nil
        yield
        60.times do
          clipboard = @mutex.synchronize { @clipboard }
          return clipboard if clipboard
          sleep 0.5
        end
        warn 'clipboard still empty after 30s'
        nil
      else
        @mutex.synchronize { @clipboard }
      end
    end
```